symfit/contrib/interactive_guess/interactive_guess.py

```diff
@@ -10,7 +10,7 @@
 import numpy as np
 from scipy.stats import gaussian_kde
 
-plt.ioff()
+#plt.ioff()
```

Comment:
Is this comment still needed? And do I actually need to do the `plt.ioff()` to make it work in a normal Python script (not notebook)?

Reply:
Its needed to be commented out for my notebook implementation to work. I'm not sure if you really need it in a normal python file. I can give it a try if I have time.